cogs/roles.py

```diff
@@ -141,6 +141,32 @@ async def inrole(self, ctx, *, queryRole):
                           editable_content=False)
             await pages.paginate()
 
+    @commands.command()
+    async def inchannel(self, ctx):
+        """Returns list of users in current channel"""
+        members = None
+        channel = ctx.message.channel
+        members = channel.members
+
+        if members is None:
+            return
+
+        channelUsers = list(map(lambda m: str(m) + "\n", members))
```

Comment:
use underscores (`channel_users`) for consistency

Reply:
Resolved in b20d0ac